lib/config/fileconf_test.go

```diff
@@ -332,6 +332,34 @@ func TestAuthenticationSection(t *testing.T) {
 	}
 }
 
+func TestAuthenticationConfig_Parse_StaticToken(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		desc  string
+		token string
+	}{
+		{"file path on windows", `C:\path\to\some\file`},
+		{"literal string", "some-literal-token"},
```

Comment:
nit: use field names, it's more readable and more resistant to future changes.

```suggestion
		{desc: "file path on windows", token: `C:\path\to\some\file`},
		{desc: "literal string", token: "some-literal-token"},
```

(Sadly, the reference I pasted for table tests doesn't do this.)